app/controllers/api/v1/accounts/articles_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
     @portal_articles = @portal.articles
     @all_articles = @portal_articles.search(list_params)
     @articles_count = @all_articles.count
-    @articles = @all_articles.page(@current_page)
+    @articles = @all_articles.page(@current_page).order_by_updated_at
```

Comment:
This should be sorted before the pagination right? CC: @tejaswinichile

Reply:
Thanks for the suggestion, i believe doing above won't hurt but yeah happy to make the changes as required CC @tejaswinichile

Reply:
I have made the changes. Sorted before pagination.